lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -27,19 +20,7 @@ export default function DashboardApp() {
       </div>
       <div className="flex-grow px-3">
         <div className="mx-auto max-w-6xl">
-          <Switch>
-            <Route path="/assignments/:assignmentId">
-              <AssignmentActivity />
-            </Route>
-            <Route path="/courses/:courseId">
-              <CourseActivity />
-            </Route>
-            <Route path="">
-              <OrganizationActivity
-                organizationPublicId={organizationPublicId}
-              />
-            </Route>
-          </Switch>
+          <ComponentWithLoaderWrapper />
```

Comment:
The standard mechanisms wouter provides for route matching, will always replace the children as soon as the route changes, so I moved to a different way to match routes via `useRoute`, which still benefits from the use of wouter.